ROADMAP.adoc

```diff
@@ -57,11 +57,12 @@ version name of a given release.
 [[minishift-110]]
 === Minishift 1.1.0
 
-- [ ] Implement caching of OpenShift image https://github.com/minishift/minishift/issues/143[#143]
+- [x] Add update command  https://github.com/minishift/minishift/issues/72[#72]
```

Comment:
I agree, probably a bit nicer in separate commit, but also some effort. But I do it just for you :-)